definitions/v7/teracod.yml

```diff
@@ -0,0 +1,186 @@
+---
+id: teracod
+name: teracod
+description: "teracod (Movie Zone) is a HUNGARIAN Private Torrent Tracker for 0DAY / GENERAL"
+language: hu-HU
+type: private
+encoding: UTF-8
+links:
+  - https://teracod.net/
+legacylinks:
+  - https://moviezone.ws/
+
+caps:
+  categorymappings:
+    - {id: 42, cat: Movies/3D, desc: "Film/3D/Hun"}
+    - {id: 19, cat: Movies/SD, desc: "Film/CAM/Eng"}
+    - {id: 16, cat: Movies/SD, desc: "Film/CAM/Hun"}
+    - {id: 36, cat: Movies/DVD, desc: "Film/DVD/Eng"}
+    - {id: 35, cat: Movies/DVD, desc: "Film/DVD/Hun"}
+    - {id: 34, cat: Movies/HD, desc: "Film/HD/Eng"}
+    - {id: 33, cat: Movies/HD, desc: "Film/HD/Hun"}
+    - {id: 32, cat: Movies/SD, desc: "Film/SD/Eng"}
+    - {id: 31, cat: Movies/SD, desc: "Film/SD/Hun"}
+    - {id: 41, cat: Movies/UHD, desc: "Film/UHD/Hun"}
+    - {id: 28, cat: PC/Games, desc: "Játék/ISO"}
+    - {id: 27, cat: Console, desc: "Játék/Konzol"}
+    - {id: 26, cat: PC/Games, desc: "Játék/RIP"}
+    - {id: 25, cat: Audio/Video, desc: "Klipp"}
+    - {id: 24, cat: Books/EBook, desc: "Könyvek/Eng"}
+    - {id: 23, cat: Books/EBook, desc: "Könyvek/Hun"}
+    - {id: 17, cat: PC/Mobile-Other, desc: "Mobile"}
+    - {id: 20, cat: PC, desc: "Program/Egyéb"}
+    - {id: 14, cat: PC/ISO, desc: "Program/ISO"}
+    - {id: 13, cat: TV/SD, desc: "Sorozat/DVDR/Eng"}
+    - {id: 12, cat: TV/SD, desc: "Sorozat/DVDR/Hun"}
+    - {id: 40, cat: TV/HD, desc: "Sorozat/HD/Eng"}
+    - {id: 39, cat: TV/HD, desc: "Sorozat/HD/Hun"}
+    - {id: 38, cat: TV/SD, desc: "Sorozat/SD/Eng"}
+    - {id: 37, cat: TV/SD, desc: "Sorozat/SD/Hun"}
+    - {id: 30, cat: XXX, desc: "XXX"}
+    - {id: 18, cat: XXX/x264, desc: "XXX/HD"}
+    - {id: 15, cat: XXX/ImageSet, desc: "XXX/Képsorozatok"}
+    - {id: 11, cat: Audio/Lossless, desc: "Zene/EN/Lossless"}
+    - {id: 29, cat: Audio/Lossless, desc: "Zene/HU/Lossless"}
+    - {id: 22, cat: Audio/MP3, desc: "Zene/Mp3/Eng"}
+    - {id: 21, cat: Audio/MP3, desc: "Zene/Mp3/Hun"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep]
```

Comment:
Why v7?

Reply:
genre

but not sure why the file isn't matching local
![image](https://user-images.githubusercontent.com/55419169/180805698-70a489c9-7888-497e-b577-efa0a6c96607.png)

Reply:
sees git finally fixed itself